pkg/install/install.go

```diff
@@ -76,9 +87,30 @@ spec:
 		return fmt.Errorf("domain dns: %w", err)
 	}
 	fmt.Println("    Domain DNS set up...")
+	fmt.Println("    Finished configuring Kourier")
 
-	fmt.Println("    Finished installing Kourier Networking layer")
+	return nil
+}
 
+// KourierMinikube runs the minikube-specific setup for Kourier
+func KourierMinikube() error {
+	fmt.Println("🕸 Configuring Kourier for Minikube...")
+
+	if err := retryingApply("https://github.com/knative/serving/releases/download/v" + servingVersion + "/serving-default-domain.yaml"); err != nil {
+		return fmt.Errorf("default domain: %w", err)
+	}
+	if err := waitForPodsReady("knative-serving"); err != nil {
+		return fmt.Errorf("core: %w", err)
+	}
+
+	fmt.Println("    Domain DNS set up...")
+
+	tunnel := exec.Command("minikube", "tunnel", "--profile", "minikube-knative")
```

Comment:
Interesting that we run `tunnel` for them I like it.
Does this process runs on the background? 
What happens when user tries `kn quickstart minikube` a second time, do we delete the minikube and do minikube start and what happens with this tunnel process? do we run it again and everything is fine?

Reply:
good point -- this should probably be part of the cluster create process in `/pkg/minikube/minikube.go` so that it won't get run a second time if the cluster already exists.